src/app/public/modules/split-view/types/split-view-message-type.ts

```diff
@@ -0,0 +1,7 @@
+export enum SkySplitViewMessageType {
+  FocusWorkspace = 0,
+  IteratorEnableNextButton = 1,
```

Comment:
Think we could mimic the `SkyFlyoutMessageType` iterator names?
https://github.com/blackbaud/skyux-flyout/blob/master/src/app/public/modules/flyout/types/flyout-message-type.ts#L4-L7

Reply:
Oops, I meant to do that. Yes, good catch.

Reply:
Done.